pom.xml

```diff
@@ -265,6 +265,8 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>3.5.2</version>
                 <configuration>
+                    <forkCount>1</forkCount>
+                    <reuseForks>false</reuseForks>
```

Comment:
Ok this is obviously not a solution, but I want to know what the reason for the test failures are, are they actual issues ore is it cross test interactions? It seems we are suffering from global setup issues due to the Singleton that is OpenFeatureApi. @toddbaert any suggestion how to get around this?

Reply:
Which tests are failing? Today we merged something which increases the stability of the spectests

Reply:
I think there's a test annotation to run some tests in isolation, which may be the best solution here. If you cannot narrow down the exact suite I'd support you adding forkCount 1 for now, and I can look into it later (or somebody else).

If you want to keep digging at it yourself, you're more than welcome, but it's not 100% related to your improvement, so I don't want to force you to do it if you are sure it's a bad test interaction.

Up to you!

Reply:
@ Execution(SAME_THREAD) annotation

Reply:
Ah thanks! Yes it is always the same tests so I'm gonna annotate them and remove this global one. And then this should be done.

Reply:
Ok so I tried this out locally and got some fixes but still have some failing tests, with annotations it is not possible to get the same isolation as with no reuse for forks. I am gonna spent more time tomorrow on this and if I can't resolve it I would suggest to move this to a followup MR?

Reply:
I'm in favor of merging as is and following up with improvements in a separate PR. @toddbaert @liran2000 any concerns about that?

Reply:
No objections.